guillotina/api/service.py

```diff
@@ -36,6 +42,30 @@ def __setattr__(self, name, value):
 
 
 class Service(View):
+    __validator__ = None
+
+    async def validate(self):
+        data = await self.request.json()
+        if(self.__class__.__validator__ is None):
+            try:
+                self.__class__.__schema__ = self.__config__['parameters'][0]['schema']['$ref'].split('/')[-1]
```

Comment:
Also, this needs to look at open api 3's way of describing request body: https://swagger.io/docs/specification/describing-request-body/